internal/gitstream/sync.go

```diff
@@ -177,6 +183,17 @@ func (s *Sync) Run(ctx context.Context) error {
 	return nil
 }
 
+func (s *Sync) getAuthorsToSkip() map[string]struct{} {
+
+	authorsToSkip := map[string]struct{}{}
+	for _, author := range s.DownstreamConfig.SkipCommitsFrom {
+		if _, ok := authorsToSkip[author]; !ok {
+			authorsToSkip[author] = struct{}{}
+		}
+	}
+	return authorsToSkip
+}
```

Comment:
In fact, I'd probably make this function more generic:
```suggestion
func (s *Sync) makeStringSet(s []string) map[string]struct{} {
	m := make(map[string]struct{}, len(s))

	for _, str := range s {
		m[str] = struct{}{}
	}

	return m
}
```

Reply:
Fixed.